this & object prototypes/foreword.md

```diff
@@ -1,21 +1,21 @@
 
-*this & Object Prototypes* is a wonderful continuation to the series. It does a great and natural job of building on the prior book, Scope & Closures, and extending that knowledge to a very important part of the JS language, the `this` keyword and prototypes. These two simple things are pivotal for what you will learn in the future books, because they are foundational to doing real programming with JavaScript. The concept of how to create objects, relate them, and extend them to represent things in your application is necessary to create large and complex applications in JavaScript. And without them, creating complex applications (such as Google Maps) wouldn’t be possible in JavaScript.
+*this & Prototipagem de Objetos* é uma maravilhosa continuação da série. Ela faz um grandioso e natural trabalho de contrução no livro anterior, Escopo & Encerramentos, e extende este conhecimento para uma parte muito importante da linguagem JS, a palavra-chave `this` e a prototipagem. Essas duas coisas simples são cruciais para o que você vai aprender nos livros futuros, porque elas são fundamentais para fazer a real programação com JavaScript. O conceito de como criar objetos, relacioná-los, e extendê-los para representar coisas na sua aplicação é necessário para criar aplicações amplas e complexas com JavaScript. E sem eles, criar aplicações complexas (como o Google Maps) não serão possíveis em JavaScript.
 
-I would say that the vast majority of web developers probably have never built a JavaScript object and just treat the language as event-binding glue between buttons and AJAX requests. I was in that camp at a point in my career, but after I learned how to master prototypes and create objects in JavaScript, a world of possibilities opened up for me. If you fall into the category of just creating event-binding glue code, this book is a must-read; if you just need a refresher, this book will be a go-to resource for you. Either way, you will not be disappointed. Trust me!
+Eu diria que a grande maioria dos desenvolvedores web provavelmente nunca tenham construído um objeto JavaScript e só trata a linguagem como cola de ligação de eventos entre botões e requisições com AJAX. Eu estava neste campo até um ponto da minha carreira, mas depois que eu aprendi a dominar os protótipos e criar objetos em JavaScript, um mundo de possibilidades se abriu para mim. Se você se enquadra nessa categoria de apenas criar código de ligação entre eventos, este livro é para uma leitura obrigatória; se você apenas precisa de uma reciclagem, este livro será um bom recurso para você. De qualquer forma, you não irá se desapontar. Confie em mim!
```

Comment:
este livro é ~~para~~ uma leitura obrigatória
...
~~you~~ **você** não irá se desapontar